### ISSUE TITLE
inconsistency in EOF (end of file newline)

### ISSUE THREAD

**Issue by kevinvalk** [jinxdash/prettier-plugin-rust#21]
### Input code

```rust
fn eof() {}
\n
```

### Output code

```rust
fn eof() {}
\ No newline at end of file
```

### Additional context

It seems that this plugin does always add an EOF (end of file newline). Specifically for functions it does not do that. If you have a file with just a comment `// test` it does make it into `// test\n`.

That said, in the following piece of code rust prettier does NOT add a newline.
```rust
fn eof() {}
// comment
```

**Comment by ZerdoX-x** [comment]
For me it even removes the empty line. I am happy that running `prettier --check` doesn't throw errors about missing EOF newline in `.rs` files :)
